CMakeLists.txt

```diff
@@ -208,13 +212,18 @@ set(NGRAPH_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR}/src/ngraph)
 
 set(EXTERNAL_INSTALL_DIR ${CMAKE_BINARY_DIR}/external)
 
-if (NGRAPH_ONNX_IMPORT_ENABLE)
-    find_package(Protobuf 2.6.1 REQUIRED)
-endif()
-
 if(NOT DEFINED EXTERNAL_PROJECTS_ROOT)
     set(EXTERNAL_PROJECTS_ROOT ${CMAKE_CURRENT_BINARY_DIR})
 endif()
+
+if (NGRAPH_ONNX_IMPORT_ENABLE)
+    if (NOT NGRAPH_USE_SYSTEM_PROTOBUF)
+        include (cmake/external_protobuf.cmake)
+    else()
+        find_package(Protobuf 2.6.1 REQUIRED)
```

Comment:
nGraph would be compatible with more models if we started to use 3.x series of Protobuf. Latest version seems to be 3.6.1, should we use that?